Source/UI/CommonUI/View/Core/ButtonView.swift

```diff
@@ -9,6 +9,12 @@ class ButtonView: UIView {
     var style: ElementButtonStyle?
     lazy var constraintLeading: NSLayoutConstraint? = buttonTextLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 120.0)
 
+    private(set) lazy var borderView: BorderView = {
+        let view = BorderView().disabledAutoresizingIntoConstraints()
+        view.backgroundColor = .clear
```

Comment:
Could we just move this background color inside BorderView ? I don't see any reason why a view designed to provide border would ever have a default background color.

Reply:
> BorderView is responsable only for borders for buttons or textfield or any UI components

Exactly. So it should not be expected for consumer to write `view.backgroundColor = .clear`.